index.html

```diff
@@ -5,27 +5,43 @@
     <meta charset="UTF-8">
     <title>Flickr-Poc</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
-    <link rel="stylesheet" type="text/css" href="bower_components/lightbox2/dist/css/lightbox.css">
+    <link rel="stylesheet" type="text/css" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
```

Comment:
Can we have a smaller carousel, that when the images are selected, open in lightbox? The hope of each phase is to build upon the previous one. So let's keep lightbox, as well as include bootstrap.